Resources/Prototypes/_EE/Entities/Objects/Weapons/Weapons/swords.yml

```
@@ -0,0 +1,50 @@
+      state: icon
+    - type: Item
+      sprite: _EE/Clothing/SolAlliance/Swords/SAN_officer_sword.rsi
+      size: Normal
+
+- type: entity
+  parent: ClothingBeltSheath
+  id: ClothingBeltSheathSANOfficerFilled
+  name: Solarian sabre sheath
+  description: An ornate sheath designed to hold an officer's blade.
+  components:
+  - type: ContainerFill
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)